src/Controllers/Version3/class-wc-rest-orders-controller.php

```diff
@@ -244,6 +244,18 @@ public function get_item_schema() {
 
 		$schema['properties']['coupon_lines']['items']['properties']['discount']['readonly'] = true;
 
+		$meta_data_item_properties_ref = &$schema['properties']['line_items']['items']['properties']['meta_data']['items']['properties'];
```

Comment:
Ideally, we can add these additional fields for both V2 and V3

Reply:
Great! With the [recent commits](https://github.com/woocommerce/woocommerce-rest-api/pull/229/files/a4db538417f62f8d7324b594f393b45424c5fa8b..d730e0fb9cd527597863a257f5bdbc8ed8d6b2bd), these fields are now available for both V2 and V3.